app/Model/CmpEnrollmentAttribute.php

```diff
@@ -131,6 +131,13 @@ public function availableAttributes() {
         'ldap_name' => 'edu_person_affiliation',
         'saml_name' => 'edu_person_affiliation'
       ),
+      'authn_authority' => array(
+        'required'  => RequiredEnum::Optional,
+        'label'     => _txt('fd.authn_authority'),
+        'env_name'  => 'CMP_EF_AUTHNAUTHORITY',
+        'ldap_name' => 'AuthenticatingAuthority',
```

Comment:
If `AuthenticatingAuthority` is not part of an ldap schema then `ldap_name` should be set `''`

Reply:
Replacing with an empty string value